### PR TITLE
Update changes to 1.6.0 for machines that switched to Rocky OS

### PR DESCRIPTION
**Pull request by RatkoVasic-NOAA**
### Summary

Since 1.6.0 release some machines switched to Rocky OS. Need to update config files for those machines.

### Testing

Install from new branch

### Applications affected

All

### Systems affected

Hera, Orion, NOAA-cloud,... (will check for others)

### Dependencies
No dependences

### Issue(s) addressed

Closes issue #1316 

### Checklist
- [x] This PR addresses one issue/problem/enhancement, or has a very good reason for not doing so.
- [x] These changes have been tested on the affected systems and applications.
- [x] All dependency PRs/issues have been resolved and this PR can be merged.

### CODE REVIEW

**Comment by climbfuji** [comment]
I am removing myself as a reviewer - this is updating a non-month old release/branch that only NOAA still uses

File: configs/sites/jet/mirrors.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 mirrors:
   local-source:
     fetch:
-      url: file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache
+      url: file:///contrib/spack-stack/source-cache
```
Comment (AlexanderRichert-NOAA) [inline configs/sites/jet/mirrors.yaml]: Can this be safely reverted?

File: configs/sites/jet/mirrors.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
       access_pair:
       - null
       - null
       access_token: null
       profile: null
       endpoint_url: null
     push:
-      url: file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache
+      url: file:///contrib/spack-stack/source-cache
```
Comment (AlexanderRichert-NOAA) [inline configs/sites/jet/mirrors.yaml]: " "

File: configs/sites/jet/mirrors.yaml (hunk 1 above)
Comment (RatkoVasic-NOAA) [reply configs/sites/jet/mirrors.yaml]: Yes, I copied today everything from current mirror to that one.

**Review by AlexanderRichert-NOAA — APPROVED** [review]
Please double check the source cache path on Jet, but otherwise this looks good to me.

**Comment by AlexanderRichert-NOAA** [comment]
@RatkoVasic-NOAA if/when you're ready please bypass the branch protection settings and merge :)

**Comment by RatkoVasic-NOAA** [comment]
> @RatkoVasic-NOAA if/when you're ready please bypass the branch protection settings and merge :)

I guess I have to wait for unit tests to finish first. How do I merge this branch (bypass protection)?

**Comment by AlexanderRichert-NOAA** [comment]
Done. It might require admin privileges, I'm not sure.

**Comment by RatkoVasic-NOAA** [comment]
> Done. It might require admin privileges, I'm not sure.

Thanks!

**Pull request merged**